DataConnectors/CiscoUmbrella/CiscoUmbrella_API_FunctionApp.json

```diff
@@ -0,0 +1,171 @@
+{
+    "id": "CiscoUmbrellaDataConnector",
+    "title": "Cisco Umbrella",
+    "publisher": "Cisco",
+    "descriptionMarkdown": "The Cisco Umbrella data connector provides the capability to ingest information about your Cisco Umbrella events stored in Amazon S3 into Azure Sentinel through the Amazon S3 REST API. [See documentation.](https://docs.umbrella.com/deployment-umbrella/docs/log-management) ",
```

Comment:
Please update this as follows. You can follow similar format for other data connectors as well. 

"The Cisco Umbrella data connector provides the capability to ingest [Cisco Umbrella](https://docs.umbrella.com/) events stored in Amazon S3 into Azure Sentinel using the Amazon S3 REST API. Refer to [Cisco Umbrella log management documentation](https://docs.umbrella.com/deployment-umbrella/docs/log-management) for more information."

Reply:
done